vanilla-extract/app/root.tsx

```diff
@@ -1,38 +1,31 @@
-import "~/global.css";
-
-import { cssBundleHref } from "@remix-run/css-bundle";
-import type { LinksFunction, MetaFunction } from "@remix-run/node";
 import {
   Links,
-  LiveReload,
   Meta,
   Outlet,
   Scripts,
   ScrollRestoration,
 } from "@remix-run/react";
 
-export const links: LinksFunction = () =>
-  cssBundleHref ? [{ rel: "stylesheet", href: cssBundleHref }] : [];
-
-export const meta: MetaFunction = () => ({
-  charset: "utf-8",
-  title: "New Remix App",
-  viewport: "width=device-width,initial-scale=1",
-});
+import "./global.css"; // FIXME
```

Comment:
@markdalgleish I'm a bit confused how to handle this file. Its contents doesn't make it to the build anyway, can you help? 🙏🏼

Reply:
It's because `**/*.css.ts` was removed from the `sideEffects` field in `package.json`. I've just pushed an update.

Reply:
Wow, I would have never figured this out, thanks!